pkg/lightning/backend/local/local.go

```diff
@@ -2435,7 +2435,12 @@ func (w *Writer) flush(ctx context.Context) error {
 func (w *Writer) Close(ctx context.Context) error {
 	defer w.kvBuffer.destroy()
 	defer w.local.localWriters.Delete(w)
-	return w.flush(ctx)
+	err := w.flush(ctx)
+	// FIXME: in theory this line is useless, but I our test with go1.15
```

Comment:
Is the behavior of GC different among different go runtime, from 1.13 ~ 1.16

Reply:
I think GO had been optimizing its GC behaviors across these versions. But sadly, In my newest benchmark with go 1.16.3, it still has this issue. The memory usage increased up to more than 75GB within 10min. 😂